ReSharper.FSharp/test/data/features/quickFixes/removeUnexpectedArguments/Function 3 - many args.fs

```diff
@@ -0,0 +1,6 @@
+module Module
+
+let f x y = x + y
+let g = 2 + ((((f 2 3) {caret}3) 4) ((39 - 80)) ({ new IDisposable with
```

Comment:
It'd be really great to try to keep things simple, so it'd be easier to see what exactly does this test tries to cover.